goodfood/goodfood/urls.py

```diff
@@ -14,8 +14,9 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import include, re_path
 
 urlpatterns = [
```

Comment:
Where is home page url?

Reply:
I have not added that if anyone wants he/she is welcome I, my work was to add a working `complain` app and it is working.